Storage/SQL/BrowserDB.swift

```diff
@@ -203,65 +203,60 @@ open class BrowserDB {
             SentryIntegration.shared.sendWithStacktrace(message: "Unable to get a transaction: \(error.localizedDescription)", tag: "BrowserDB", severity: .error)
 
             // Check if the error we got is recoverable (e.g. SQLITE_BUSY, SQLITE_LOCK, SQLITE_FULL).
-            // If so, we *shouldn't* move the database file to a backup location and re-create it.
-            // Instead, just crash so that we don't lose any data.
+            // If so, we will try moving the database file to a backup location and copy it back to
+            // its original location. This should effectively "unlock" the database file.
```

Comment:
> The `SQLITE_LOCKED` result code indicates that a write operation could not continue because of a conflict within the same database connection or a conflict with a different database connection that uses a shared cache. 

Shared cache mode is disabled by default, so it's a surprise if you got `SQLITE_LOCKED`.